clustermesh/clustermesh.go

```diff
@@ -905,6 +905,16 @@ func (k *K8sClusterMesh) Connect(ctx context.Context) error {
 			aiLocal.ClusterName, aiLocal.ClusterID)
 	}
 
+	cid, err := strconv.Atoi(aiRemote.ClusterID)
+	if err != nil {
+		k.Log("❌ Cluster has the non-numeric ID 0 (cluster ID: %s): %s", aiLocal.ClusterID, err)
```

Comment:
I'm not entirely sure this message is correct wrt. ID 0. I think this could be put a bit more succinctly:
 
```suggestion
		k.Log("❌ Cluster has non-numeric cluster ID 0 %s. Only numeric values 1-255 are allowed", aiRemote.ClusterID)
```